presto-orc/src/main/java/com/facebook/presto/orc/reader/NullWrappingColumnReader.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.reader;
+
+import com.facebook.presto.orc.QualifyingSet;
+import com.facebook.presto.orc.stream.BooleanInputStream;
+import com.facebook.presto.orc.stream.LongInputStream;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static com.google.common.base.Verify.verify;
+
+abstract class NullWrappingColumnReader
+        extends ColumnReader
+        implements StreamReader
+{
+    QualifyingSet innerQualifyingSet;
```

Comment:
All fields must have access modifiers. Currently, these fields appear to be `protected`, however, it is better to make make them private and add protected getters as needed.